streampark-common/src/main/scala/org/apache/streampark/common/conf/ConfigConst.scala

```diff
@@ -48,6 +48,8 @@ object ConfigConst {
 
   val KEY_JOB_ID = "jobId"
 
+  val KEY_FLINK_JOB_ID = "flinkJobId"
```

Comment:
hi MonsterChenzhuo:
I recommend to add `id` and `jobId` field in `SubmitRequest`, Don't add it to the global constant